bc_obps/service/tests/test_operation_service_v2.py

```diff
@@ -128,7 +103,7 @@ def test_assigns_single_selected_purpose():
             process_flow_diagram=MOCK_DATA_URL,
             boundary_map=MOCK_DATA_URL,
         )
-        OperationServiceV2.create_or_update_operation_v2(approved_user_operator.user.user_guid, payload, operation.id)
```

Comment:
Some of the create_or_update_operation_v2 tests I moved to separate create/update tests. If the test was more about registration, I set it up to test that function instead (which uses either create or update operation)